_benchmark/tutorials/sigv4.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: SigV4 Support
+nav_order: 70
+parent: Tutorials
+---
+
+# Running OpenSearch Benchmark with AWS Sigv4 
+
+OpenSearch Benchmark supports AWS Signature Version 4 (Sigv4) authentication. To run with Sigv4, use the following steps:
+
+1. Set up an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html) or [Role ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns) and offer it access to your OpenSearch cluster that uses SigV4 Authentication.
```

Comment:
On second thought, let's remove ` or [Role ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-arns)`. This is because users have tried using a role and it didn't work out as easily. Until we figure out a better approach for IAM roles, we don't have to include this